packages/SwingSet/bin/extract-transcript-from-kerneldb.js

```diff
@@ -1,3 +1,5 @@
+// XXX this is wrong; it needs to use the swingstore instead of opening the LMDB
+// file directly, then use stream store reads to get the transcript entries.
```

Comment:
tcket-to-self in #3206

Reply:
Anecdotally, I've got several other tools that need to run in the SES environment (usually because they use swingstore!) and debugging hasn't been particularly problematic (at least, no more so that it ever is).